### PR TITLE
Update pygfx to 0.1.13

### LINKED ISSUE #94: Update pygfx renderer
The latest (as of yet unreleased) version of pygfx changes the api of objects a bit. The whole `linalg` submodule is removed which requires the PygfxRenderer to be updated.

### PR DESCRIPTION
**Pull request by berendkleinhaneveld**
Closes #94 .

Updates pygfx to 0.1.13 which drops the `pygfx.linalg` module in favor of [pylinalg](https://github.com/pygfx/pylinalg).

### CODE REVIEW

File: tests/test_pygfx_renderer.py [hunk 1]
```diff
@@ -49,20 +49,28 @@ def test_pygfx_insert_remove():
...
+    def equals(a, b):
+        if len(a) != len(b):
+            return False
+        for x, y in zip(a, b):
+            if x != y:
+                return False
+        return True
+
     renderer.insert(child1, parent)
-    assert parent.children == (child1,)
```
Comment (Korijn) [inline tests/test_pygfx_renderer.py:53]: Why doesn't this work anymore?

Comment (berendkleinhaneveld) [reply tests/test_pygfx_renderer.py:53]: Because children is now a list instead of a tuple

Comment (berendkleinhaneveld) [reply tests/test_pygfx_renderer.py:53]: I decided to add the `equals` function here, such that if pygfx would ever change the type again, it wouldn't break this test.

**Review by Korijn — APPROVED** [review]

**Comment by berendkleinhaneveld** [comment]
I've pulled this code into #118 .

**Pull request closed without merging**